File: src/main/java/me/rsman/BetterMinecraftCore/Listeners/PlayerConnectListener.java
package me.rsman.BetterMinecraftCore.Listeners;

import me.rsman.BetterMinecraftCore.Managers.PlayerManager;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;
import org.bukkit.event.player.PlayerQuitEvent;

public class PlayerConnectListener implements Listener {

    @EventHandler
    public void onConnect(PlayerJoinEvent event){
        PlayerManager.getBaseAttributes(event.getPlayer().getUniqueId().toString(), true);
        PlayerManager.alterPlayerAttributesWithEquippedStuff(event.getPlayer());
    }

    @EventHandler
    public void onDisconnect(PlayerQuitEvent event){
        PlayerManager.playersAttributes.remove(event.getPlayer().getUniqueId().toString());
    }
}

File: src/main/kotlin/fr/rsman/betterMinecraftCore/listeners/PlayerConnectListener.kt
package fr.rsman.betterMinecraftCore.listeners

import fr.rsman.betterMinecraftCore.managers.PlayerManager
import org.bukkit.event.player.PlayerJoinEvent
import org.bukkit.event.player.PlayerQuitEvent
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener

class PlayerConnectListener : Listener {
    @EventHandler
    fun onConnect(event: PlayerJoinEvent) {
        PlayerManager.getBaseAttributes(event.player.uniqueId.toString(), true)
        PlayerManager.alterPlayerAttributesWithEquippedStuff(event.player)
    }

    @EventHandler
    fun onDisconnect(event: PlayerQuitEvent) {
        PlayerManager.playersAttributes.remove(event.player.uniqueId.toString())
    }
}